tools/seriesuid2fileid

```diff
@@ -1,4 +1,37 @@
 #!/usr/bin/perl
+
+=pod
+
+=head1 NAME
+
+seriesuid2fileid -- a script that displays relevant information and tracks the
+status of an acquired imaging series based on the series UID as an input
+argument (provided as a STDIN).
+
+=head1 SYNOPSIS
+
+perl tools/seriesuid2fileid
```

Comment:
Shouldn't the script be called seriesuid2fileid.pl? Missing the .pl extension. Maybe you can fix this in this PR?

Reply:
as agreed, we will do this separately (RM#13755)